tools/lumpy/lumpy.xml

```diff
@@ -9,64 +9,53 @@
         <exit_code range="1:" level="fatal" description="Tool exception" />
     </stdio>
     <command detect_errors="exit_code"><![CDATA[
+        #import re
+        #set one_sample_bam = re.sub('[^\w\-]', '_', str($analysis_type.input_file.element_identifier))
         #if $analysis_type.analysis_type_list == "one_sample":
-            ln -f -s '$analysis_type.input_file' input.bam &&
+            ln -f -s '$analysis_type.input_file' '$one_sample_bam' &&
         #else:
-            ln -f -s '$analysis_type.input_file' input.A.bam &&
-            ln -f -s '$analysis_type.input_fileB' input.B.bam &&
+            #set sample_a_bam = re.sub('[^\w\-]', '_', str($analysis_type.input_file.element_identifier))
+            #set sample_b_bam = re.sub('[^\w\-]', '_', str($analysis_type.input_fileB.element_identifier))
+            #if $sample_a_bam == $sample_b_bam:
+                #set sample_a_bam = "%s_a" % str($sample_a_bam)
+                #set sample_b_bam = "%s_b" % str($sample_b_bam)
+            #end if
```

Comment:
Correct me if I am wrong: if $sample_a_bam == $sample_b_bam in this block, the user made a mistake (input_file.element_identifier and input_fileB.element_identifier received the same value). I am not saying that we should not cope with that, but if on the other hand, the purpose of this is a twist to get the test for 2 samples running with a single test file, this is weird ! Please explain. On a side note, for my knowledge of cheetah/python: the str() fonction is required ? I thought it was the default type in this case.